### DISCUSSION TITLE
Call for Agenda Items: Gordian Developer Zoom Call April 5th 4pm PDT

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Gordian-Developer-Community#106] [General & Announcements]
Our next [regular Zoom call](https://www.google.com/url?q=https://us02web.zoom.us/j/81330446533?pwd%3DenlabktDMVpIUE9PcmU5SkVlU3liQT09&sa=D&source=calendar&usd=2&usg=AOvVaw02rPjcOVms5f6k_3N41Gwt) will be on April 5th at 4pm PDT.

Please reply here if you have an agenda item for the community to discuss during this call.

**Comment by simonratner** [comment]
JavaCard port of SSKR here: https://github.com/proxyco/jc-sskr. Basically, you can do SSKR on a JavaCard-based smart card, to back up / recover a secret that otherwise exists only in the card's secure element.

Let me know if anyone is interested in hearing more, or seeing a quick demo.

**Comment by simonratner** [comment]
Related to [this](https://github.com/BlockchainCommons/Gordian-Developer-Community/discussions/106#discussioncomment-5470772), using NFC in general to communicate between devices. Android and iOS both have good NFC support to interact with smart cards or tags, and in particular to talk to secure wallet hardware (whether that is in a smart card form factor, or another NFC-enabled device).

**Reply by ChristopherA** [reply]
I would really like to see an open equivalent to Coinkite's tapsigner. As you pointed out, the "NXP JCOP3 J3H145 Java Card 3.0.4 Dual-Interface SCP03" cards are inexpensive and there are multiple sources. We could also add SSKR support to recover shares or shard them. There is some experimental support now in Gordian SeedTool for iOS for saving SSKR shares to very cheap NDEF cards now.

**Comment by Fonta1n3** [comment]
I'd like to demo something I have been working on which is possibly an iteration of QuickConnect. It's currently called nostrnode because it leverages nostr architecture to hole punch to your node. Here is the [macOS repo](https://github.com/Fonta1n3/nostrnode-macOS) as proof of concept.
